projects/refer-and-monitor-and-delius/build.gradle.kts

```diff
@@ -14,14 +17,17 @@ dependencies {
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation("org.springframework.boot:spring-boot-starter-validation")
     implementation("org.springframework.boot:spring-boot-starter-web")
+    implementation("org.springframework.boot:spring-boot-starter-data-ldap")
```

Comment:
You'll need to update the values.yml file too with the LDAP URL and credentials.